Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -181,8 +183,51 @@ enum PackageShow {
                         source: SiteURL.package(.value(model.repositoryOwner),
                                                 .value(model.repositoryName),
                                                 .readme).relativeURL(),
-                        // Until the content is loaded, substitute a spinner.
-                        .spinner()
+                        .group(
+                            .class("package_tab"),
+                            // Until the content is loaded, substitute a spinner.
+                            .spinner()
+                        )
+            )
+        }
+        
+        func releaseSection() -> Node<HTML.BodyContext> {
+            .turboFrame(id: "releases",
+                        source: SiteURL.package(.value(model.repositoryOwner),
+                                                .value(model.repositoryName),
+                                                .releases).relativeURL(),
+                        .group(
+                            .class("package_tab"),
+                            // Until the content is loaded, substitute a spinner.
+                            .spinner()
+                        )
+            )
+        }
+        
+        func tabBar() -> Node<HTML.BodyContext> {
+            .group(
+                .hr(
+                    .class("minor")
+                ),
+                .section(
+                    .class("package_tabs"),
+                    .ul(
+                        .li(
+                            .a(
+                                .href("#readme"),
+                                .class("package_tab_link"),
```

Comment:
I don't think we'll need this class name if we write the JS using `querySelectorAll`.